spec/01-abstract.md

```diff
@@ -1,19 +1,22 @@
 ## Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology used by tracing tools to follow, analyze and debug a trace across multiple software components. Typically, a trace traverses more than one component which requires it to be uniquely identifiable across all participating systems. Passing along this unique identification is referred to as *context propagation*.

```

Comment:
> analyze and debug a trace

Should still be "analyze and debug a transaction" or "analyze or debug a request". I'd recommend leaving the original paragraph.

Reply:
I changed the wording to transaction.